eyieko/landville-frontend
# #167416945 Deposit with card & PIN

## Description ##
- Enable user make a deposit with card and PIN

## Type of change ##
Please select the relevant option

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes

## Checklist: ##

## How to test this PR
- Log in with the right crdentials
- From the `Make payment` dropdown button on the `home` page navbar, select `Card with PIN`
- In the resulting form, selected suitable expiry month and year and submit the form. Use `5531886652142950` as the card number. You may use any PIN and cvv
- Note that we must use the supplied card number since we are still testing. Note also that we cannot select the `Buying` option yet as the feature is not yet implemented.
- In the resulting page, enter `12345` as the OTP and submit.


## Related Pivotal Tracker stories ##
[#167416945](https://www.pivotaltracker.com/story/show/167416945)
